assets/app/views/_overview-deployment.html

```diff
@@ -72,22 +74,27 @@ <h3 class="h4">{{rc.metadata.name}}</h3>
     </div>
   </div>
 
-  <div class="pod-block">
-    <div class="component-label">Pod Template</div>
+  <div class="pod-block row">
+    <div class="col-xs-12 col-lg-6">
+      <div class="component-label">Pod Template</div>
+      <pod-template
+        pod-template="rc.spec.template"
+        images-by-docker-reference="imagesByDockerReference"
+        builds="builds">
+      </pod-template>
+    </div>
 
-    <pod-template
-      pod-template="rc.spec.template"
-      images-by-docker-reference="imagesByDockerReference"
-      builds="builds">
-    </pod-template>
+    <div class="col-xs-12 col-lg-6">
+      <div class="component-label">
+        <a href="javascript:;" ng-click="viewPodsForDeployment(rc)">
```

Comment:
Hm, if it's a problem, we have it in other places :(

Angular does document the empty href:

https://docs.angularjs.org/api/ng/directive/a